catalogs/catalog-lakehouse-paimon/src/test/java/org/apache/gravitino/catalog/lakehouse/paimon/integration/test/CatalogPaimonS3IT.java

```diff
@@ -0,0 +1,140 @@
+
+import com.google.common.collect.Maps;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import org.apache.gravitino.catalog.lakehouse.paimon.PaimonCatalogPropertiesMetadata;
+import org.apache.gravitino.catalog.lakehouse.paimon.filesystem.s3.PaimonS3FileSystemConfig;
+import org.apache.gravitino.integration.test.util.ITUtils;
+import org.apache.gravitino.integration.test.util.JdbcDriverDownloader;
+import org.apache.spark.sql.SparkSession;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.TestInstance;
+import org.testcontainers.containers.Container;
+import org.testcontainers.containers.localstack.LocalStackContainer;
+import org.testcontainers.shaded.org.awaitility.Awaitility;
+import org.testcontainers.utility.DockerImageName;
+
+@Tag("gravitino-docker-test")
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class CatalogPaimonS3IT extends CatalogPaimonBaseIT {
+
+  // private static final String S3_BUCKET_NAME =
+  // GravitinoITUtils.genRandomName("paimon-s3-bucket-");
+  private static final String S3_BUCKET_NAME = "my-test-bucket";
+  private static LocalStackContainer localStackContainer;
+  private String accessKey;
+  private String secretKey;
+  private String endpoint;
+
+  private static final String PAIMON_S3_JAR_URL =
+      "https://repo1.maven.org/maven2/org/apache/paimon/paimon-s3/0.8.0/paimon-s3-0.8.0.jar";
+
+  @Override
+  protected Map<String, String> initPaimonCatalogProperties() {
+
+    Map<String, String> catalogProperties = Maps.newHashMap();
+    catalogProperties.put("key1", "val1");
+    catalogProperties.put("key2", "val2");
+
+    TYPE = "filesystem";
+    WAREHOUSE = "s3://" + S3_BUCKET_NAME + "/";
+
+    accessKey = localStackContainer.getAccessKey();
+    secretKey = localStackContainer.getSecretKey();
+    endpoint = localStackContainer.getEndpointOverride(S3).toString();
+
+    catalogProperties.put(PaimonCatalogPropertiesMetadata.GRAVITINO_CATALOG_BACKEND, TYPE);
+    catalogProperties.put(PaimonCatalogPropertiesMetadata.WAREHOUSE, WAREHOUSE);
+    catalogProperties.put(PaimonS3FileSystemConfig.S3_ACCESS_KEY, accessKey);
+    catalogProperties.put(PaimonS3FileSystemConfig.S3_SECRET_KEY, secretKey);
+    catalogProperties.put(PaimonS3FileSystemConfig.S3_ENDPOINT, endpoint);
+
+    // Need to download the S3 dependency in the deploy mode.
+    downloadS3Dependency();
+
+    return catalogProperties;
+  }
+
+  private void downloadS3Dependency() {
+    String gravitinoHome = System.getenv("GRAVITINO_HOME");
+    try {
+      if (!ITUtils.EMBEDDED_TEST_MODE.equals(testMode)) {
+        String serverPath = ITUtils.joinPath(gravitinoHome, "libs");
+        String paimonCatalogPath =
+            ITUtils.joinPath(gravitinoHome, "catalogs", "lakehouse-paimon", "libs");
+        JdbcDriverDownloader.downloadJdbcDriver(PAIMON_S3_JAR_URL, serverPath, paimonCatalogPath);
```

Comment:
> `JdbcDriverDownloader.downloadJdbcDriver` 

Should we adjust the method name to make it seem compatible with downloading any jar?

Reply:
Yeah, you are right, I need to rename it.